<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewFrame</class>
 <widget class="QDialog" name="viewFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="background" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>801</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#background{
background-color: #a2b5b0;}</string>
   </property>
   <widget class="QWidget" name="waterWgt" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>500</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#waterWgt{border-radius: 15px;
background-color: #C5DCD6;}</string>
    </property>
    <widget class="QLabel" name="waterLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: &quot;Lato&quot;;
font-size: 26px;</string>
     </property>
     <property name="text">
      <string>Last watered</string>
     </property>
    </widget>
    <widget class="QLabel" name="daysLabel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>140</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: &quot;Lato&quot;;
font-size: 26px;</string>
     </property>
     <property name="text">
      <string>15 days ago</string>
     </property>
    </widget>
    <widget class="QLabel" name="schedDaysLabel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>210</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: &quot;Lato&quot;;
font-size: 26px;</string>
     </property>
     <property name="text">
      <string>15 days</string>
     </property>
    </widget>
    <widget class="QLabel" name="birthdayLabel">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>280</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: &quot;Lato&quot;;
font-size: 26px;</string>
     </property>
     <property name="text">
      <string>December 31, 2022</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>501</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #a2e1bc;
border-top-left-radius: 15px;
border-top-right-radius: 15px;
border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;
</string>
     </property>
     <widget class="QLabel" name="nameLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>481</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lato</family>
        <pointsize>24</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: &quot;Lato&quot;;
</string>
      </property>
      <property name="text">
       <string>PLANT NAME</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="varietyLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>481</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lato Light</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>VARIETY NAME</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="schedLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: &quot;Lato&quot;;
font-size: 26px;</string>
     </property>
     <property name="text">
      <string>Water every</string>
     </property>
    </widget>
    <widget class="QLabel" name="bdayLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>280</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: &quot;Lato&quot;;
font-size: 26px;</string>
     </property>
     <property name="text">
      <string>Growing since</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="backBtn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>81</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
font: &quot;Lato&quot;;
font-size: 64px;
border-radius: 5px;
color: #ebebeb;
}
</string>
    </property>
    <property name="text">
     <string>‚Üê</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editBtn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
font: &quot;Lato&quot;;
border: 4px solid '#bbd1cb';
border-radius: 10px;
font-size: 26px;
background-color: #bbd1cb
}
*:hover{
border: 4px solid '#b4c8c2';
background-color:'#b4c8c2';
}
</string>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
